index.html

```diff
@@ -0,0 +1,11 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+    <meta charset="UTF-8">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <title>mi pagina</title>
+</head>
+<body>
+    <h1>hola mundo</h1>
```

Comment:
Poner en mayúsculas el inicio de cada texto
```suggestion
    <title>Mi pagina</title>
</head>
<body>
    <h1>Hola mundo</h1>
```